documentation/modules/configuring/proc-loading-config-with-provider.adoc

```diff
@@ -0,0 +1,158 @@
+// Module included in the following assemblies:
+//
+// assembly-config-kafka-connect.adoc
+
+[id='proc-loading-config-with-provider-{context}']
+= Loading configuration values from external sources
+
+[role="_abstract"]
+Use the _Kubernetes Configuration Provider_ plugin to load configuration data from external sources.
+Load data from Kubernetes Secrets or ConfigMaps.
+
+Suppose you have a Secret that's managed outside the Kafka namespace, or outside the Kafka cluster.
+The provider allows you to reference the values of the Secret in your configuration without extracting the files.
+You just need to tell the provider what Secret to use and provide access rights.
+The provider loads the data without needing to restart the Kafka component, even when using a new Secret or ConfigMap.
+This capability avoids disruption when sharing a Kafka Connect instance with multiple connectors.
```

Comment:
This is a weird way of putting it. "...when a single Kafka connest instance hosts multiple connectors."

Reply:
Better. Changed